server.py

```diff
@@ -29,9 +29,13 @@ def ack(ru_ref, exercise_sid):
     data = request.get_data()
 
     tree = ET.fromstring(data)
-
     ns = {'receipt': 'http://ns.ons.gov.uk/namespaces/resources/receipt'}
 
+    tx = tree.find('receipt:tx_id', ns)
+    tx_id = tx.text
+    if tx_id:
+        app.logger.debug("Confirmed receipt for: tx_id {0}".format(tx_id))
```

Comment:
It's a bit confusing in the log output now as we output two "Confirmed receipt for .." lines. As it's a mock service it might be better to just log out the full xml that is received?